packages/lesswrong/server/authenticationMiddlewares.ts

```diff
@@ -124,20 +172,28 @@ export const addAuthMiddlewares = (addConnectHandler) => {
   addConnectHandler('/logout', (req, res, next) => {
     passport.authenticate('custom', (err, user, info) => {
       if (err) return next(err)
-      if (!user) return next()
       req.logOut()
 
-      // The accepted way to delete a cookie is to set an expiration date in the past.
-      if (getCookieFromReq(req, "meteor_login_token")) {
-        res.setHeader("Set-Cookie", `meteor_login_token= ; expires=${new Date(0).toUTCString()};`)
+      // Remove session cookies
+      const cookieUpdates = ['meteor_login_token', 'loginToken', 'connect.sid']
+        .filter(cookieName => getCookieFromReq(req, cookieName))
+        // The accepted way to delete a cookie is to set an expiration date in the past.
+        .map(cookieName => `${cookieName}= ; expires=${new Date(0).toUTCString()}`)
+      if (cookieUpdates.length) {
+        res.setHeader('Set-Cookie', cookieUpdates)
```

Comment:
I think you might need to send this as multiple updates, from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie):

> The `Set-Cookie` HTTP response header is used to send a cookie from the server to the user agent, so the user agent can send it back to the server later. **To send multiple cookies, multiple `Set-Cookie` headers should be sent in the same response.**

Reply:
Unintuitively, that would overwrite the headers (causing me a bug before). To set mutliple headers of the same name, we need to pass an array. I'll make a comment.

https://nodejs.org/api/http.html#http_request_setheader_name_value